gossip/src/crds_value.rs

```diff
@@ -415,10 +415,10 @@ impl Version {
 
 #[derive(Clone, Debug, PartialEq, Eq, AbiExample, Deserialize, Serialize)]
 pub struct NodeInstance {
-    from: Pubkey,
-    wallclock: u64,
-    timestamp: u64, // Timestamp when the instance was created.
-    token: u64,     // Randomly generated value at node instantiation.
+    pub(crate) from: Pubkey,
+    pub(crate) wallclock: u64,
+    pub(crate) timestamp: u64, // Timestamp when the instance was created.
+    pub(crate) token: u64,     // Randomly generated value at node instantiation.
```

Comment:
could avoid relaxing visibility by instead moving the datapoint submission to a helper on `NodeInstance`

Reply:
Agreed with Trent. I also like using helpers for metrics in general to reduce clutter in the important logic.